### DISCUSSION TITLE
Welcome to manim-data-structures Discussions!

### DISCUSSION THREAD

**Discussion by drageelr** [drageelr/manim-data-structures#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself.

**Comment by ZoraizQ** [comment]
Absolutely amazing initiative with great potential. A generalized Manim wrapper for streamlined data stucture animations. The examples presented in documentation clearly show how extremely useful and intuitive this tool is - from technical courses to YouTube tutorials. Following and looking forward to contribute.

**Comment by safeanimal** [comment]
Thank you for your contribution. It is extremely useful in ds teaching.
Will all the data structures be implemented in the future?

**Reply by safeanimal** [reply]
Waiting for more updating!

**Reply by drageelr** [reply]
Yes, I plan on implementing almost all Data Structures. I'll upload a Roadmap soon to the repo as well, have been quite busy with my day job so couldn't get time to work on it.

The next planned update will include 2D arrays, stack and queue. Tentatively it will be out in two or max three weeks.

**Reply by safeanimal** [reply]
Exciting! I will defenitely use it for my future data structures presentation and recommend it to other people having similar demand.

**Reply by drageelr** [reply]
Thank you @safeanimal for your feedback. Really appreciate it!
